client/src/redux/sagas/fixtures/notificationsJobsEvents.fixture.js

```diff
@@ -0,0 +1,205 @@
+export const jobStatusUpdatePending = {
+  event_type: 'job',
+  datetime: '1600604729',
```

Comment:
This timestamp - is it epoch seconds? Should it be an ISO timestamp instead? I'm thinking of the difference between Javascript epoch milliseconds vs unixtime epoch seconds. If necessary we can modify that backend to return an ISO maybe. Worth discussing consistency with larger group, too, since this is coming up with Aloe.

Reply:
@jchuahtacc , @rstijerina ,  so maybe a follow up ticket for this in backend epic is needed.

Reply:
If we're not reading this timestamp in the client it probably doesn't matter